packages/clay-navigation-bar/src/index.tsx

```diff
@@ -64,6 +73,7 @@ function ClayNavigationBar({
 	fluidSize,
 	inverted = false,
 	itemAriaCurrent: ariaCurrent = true,
+	messages,
```

Comment:
Just to avoid the problem of breaking changes in components, we can add a default value here. This way, whoever is using it will not see the application build breaking.

```suggestion
	messages = {
		close: 'Close',
		open: 'Open',
		trigger: '{0} Menu, Current Page: {1}',
	}
```